Packs/FeedCSV/Integrations/FeedCSV/FeedCSV.yml

```diff
@@ -61,6 +61,12 @@ configuration:
   required: false
   type: 8
   defaultvalue: ""
+- additionalinfo: If selected, the indicator type will be auto-detected for each indicator.
+  defaultvalue: 'true'
+  display: Auto-detect indicator type
```

Comment:
@orhovy I think you are should apply andrew's comments to all 3 feeds.

Reply:
That is exactly what I have done, the only one I did not change is JsonFeed which had already auto-detect which is in master

Reply:
you are right, I did not see the `-`.